unit/Makefile

```diff
@@ -16,14 +17,14 @@ LIBS += ../src/ansi-c/ansi-c$(LIBEXT) \
         ../src/cpp/cpp$(LIBEXT) \
         ../src/json/json$(LIBEXT) \
         ../src/linking/linking$(LIBEXT) \
-        ../src/util/util$(LIBEXT) \
-        ../src/big-int/big-int$(LIBEXT) \
-        ../src/goto-programs/goto-programs$(LIBEXT) \
         ../src/pointer-analysis/pointer-analysis$(LIBEXT) \
         ../src/langapi/langapi$(LIBEXT) \
         ../src/assembler/assembler$(LIBEXT) \
         ../src/analyses/analyses$(LIBEXT) \
+        ../src/goto-programs/goto-programs$(LIBEXT) \
         ../src/solvers/solvers$(LIBEXT) \
+        ../src/util/util$(LIBEXT) \
+        ../src/big-int/big-int$(LIBEXT) \
```

Comment:
For the awful reason that the order effects whether it compiles on GCC: https://stackoverflow.com/questions/45135/why-does-the-order-in-which-libraries-are-linked-sometimes-cause-errors-in-gcc - 

From memory the `analyses.a` depends on the `goto-programs.a` and lots depends on `util.a` which depends on `big-int.a`. I'm not even sure this will be the last rearrangement. I would guess as the first unit tests are written for each library are added, these will fluctuate about.